source/Makefile

```diff
@@ -17,3 +17,9 @@ HAS_JSYAML:
 	    echo 'Error: need "npm install -g js-yaml"' 1>&2; \
 	    exit 1;                                           \
 	fi
+
+HAS_AJV:
+	@if ! command -v ajv > /dev/null; then            \
+	    echo 'Error: need "npm install -g ajv"' 1>&2; \
```

Comment:
Thanks for pointing this out. It's presently not used because I haven't translated the test format to a JSON schema, and would prefer not to do that until we've settled on a format.

It may make sense to defer JSON schema creation to follow-up work once the syntax is approved and POCs are complete. I don't think we'll actually need it until the effort to port over existing spec tests commences.